scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: hosts
+  gather_facts: False
+  vars:
+    caasp_image_url: "http://provo-clouddata.cloud.suse.de/images/openstack/x86_64/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    image_name: "SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud"
+    stack_name: "caasp-stack"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{ caasp_image_url }}"
+      dest: "{{ caasp_image_path }}"
+      mode: 0600
+    register: _download_caasp_image
+    until: _download_caasp_image | success
+    retries: 5
+    delay: 2
+
+  - name: Upload CaaSP image to glance
+    always_run: True
+    shell: |
+      source ~ardana/service.osrc
+      if ! openstack image list | grep "{{ image_name }}"; then openstack image create --public --disk-format qcow2 --container-format bare --file "{{ caasp_image_path }}" "{{ image_name }}"; fi
+    args:
+      executable: /bin/bash
+
+  - name: Create heat stack for CaaSP deployment
+    always_run: True
+    shell: |
+      source ~ardana/service.osrc
+      openstack stack create -t caasp-stack.yaml -e caasp-environment.yaml --parameter image="{{ image_name }}" "{{ stack_name }}"
+      while openstack stack show "{{ stack_name }}" | grep -q CREATE_IN_PROGRESS; do echo "CaaSP stack creation in progress"; done
+      result=$(openstack stack show caasp-stack | grep -w stack_status | awk '{print $4}')
+      if [ $result = 'CREATE_COMPLETE' ]; then echo "CaaSP stack created successfully."; else echo "Failed to create stack: $result" exit 1; fi
```

Comment:
Small typo that causes the ansible task to actually finish successfully instead of failing when the heat stack creation fails: `else echo "Failed to create stack: $result" exit 1;` should be `else echo "Failed to create stack: $result"; exit 1;`